tests/publiccloud/azure_aitl.pm

```diff
@@ -0,0 +1,133 @@
+# SUSE's openQA tests
+#
+# Copyright 2021-2024 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in Azure using azure-cli binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use mmapi 'get_current_job_id';
+use utils qw(zypper_call script_retry);
+use version_utils 'is_sle';
+use registration qw(add_suseconnect_product get_addon_fullname);
+use JSON;
+use XML::LibXML;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    # If 'az' is preinstalled, we test that version
+    if (script_run("which az") != 0) {
+        add_suseconnect_product(get_addon_fullname('pcm'), (is_sle('=12-sp5') ? '12' : undef));
+        add_suseconnect_product(get_addon_fullname('phub')) if is_sle('=12-sp5');
+        zypper_call('in azure-cli jq python311 python3-susepubliccloudinfo ');
+    }
+    assert_script_run('az version');
+
+    my $provider = $self->provider_factory();
+
+    my $image_url = get_required_var('PUBLIC_CLOUD_IMAGE_LOCATION');
+    my $region = get_var('PUBLIC_CLOUD_REGION', 'westeurope');
```

Comment:
```suggestion
    my $region = get_var('PUBLIC_CLOUD_REGION');
```

this is design decision done on purpose . It is responsibility of triggering logic / job group settings to define in which region test will run . And if region is not defined test MUST fail

Reply:
Sure, but in the specific instance for AITL, it kind of doesn't matter. 

I couldn't get AITL jobs running anywhere but US (westus1,2,3 and etc.) so for the time being, the region is hardcoded.

Reply:
We also do this: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7dab163d5ce61f9cbd3047677c3ad29c92eab956/lib/publiccloud/k8sbasetest.pm#L91 So if nothing is specified it will be set to `westeurope` anyway :shrug:

Reply:
so why you need this variable at all ? just drop it